Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -710,12 +736,26 @@ theorem Monic.isUnit_leadingCoeff_of_dvd {a p : R[X]} (hp : Monic p) (hap : a 
     IsUnit a.leadingCoeff :=
   isUnit_of_dvd_one (by simpa only [hp.leadingCoeff] using leadingCoeff_dvd_leadingCoeff hap)
 
+theorem Irreducible.roots_card_le_one (hirr : Irreducible p) : p.roots.card ≤ 1 := by
```

Comment:
Is it intentional that this is in the `Polynomial.Irreducible` namespace? I'm not sure if this is a good choice.